packages/core/src/percy.js

```diff
@@ -321,6 +325,18 @@ export class Percy {
     }.call(this));
   }
 
+  // Driver Wrapper
+  async driverWrapper(options) {
+    this.sessionId = options.sessionId;
+    this.commandExecutorUrl = options.commandExecutorUrl;
+    this.capabilities = options.capabilities;
+    this.snapshotName = options.snapshotName;
+    this.sessionCapabilites = options.sessionCapabilites;
+    const provider = ProviderResolver.resolve(this.sessionId, this.commandExecutorUrl, this.capabilities, this.snapshotName, this.sessionCapabilites);
```

Comment:
To big parameters may be we can pass the few of them as `opts = {}` and details can be extracted there? May be snapshot name can be outside opts. Just a suggestion.
```suggestion
    opts = {
        snapshotName: this.snapshotName, 
        capabilities: this.capabilities, 
        sessionCapabilities: this.sessionCapabilites
    }
    const provider = ProviderResolver.resolve(this.sessionId, this.commandExecutorUrl, opts);
```

Reply:
Moving `snapshotName` out of constructor.

Reply:
Done.